Lib/fontTools/feaLib/ast.py

```diff
@@ -1256,23 +1256,29 @@ def build(self, builder):
         """Calls the builder object's ``add_multiple_subst`` callback."""
         prefix = [p.glyphSet() for p in self.prefix]
         suffix = [s.glyphSet() for s in self.suffix]
-        if not self.replacement and hasattr(self.glyph, "glyphSet"):
-            for glyph in self.glyph.glyphSet():
-                builder.add_multiple_subst(
-                    self.location,
-                    prefix,
-                    glyph,
-                    suffix,
-                    self.replacement,
-                    self.forceChain,
-                )
+        if hasattr(self.glyph, "glyphSet"):
+            originals = self.glyph.glyphSet()
         else:
+            originals = [self.glyph]
```

Comment:
codecov says this line isn't covered, should it be?

Reply:
I’m not sure. Going from the parser, this should never happen, but the old code has a similar check so I guessed it would be for constructing the ast classes directly so one can use strings for glyph names instead of `ast.GlyphName()`.